src/hotspot/cpu/aarch64/c1_LIRAssembler_aarch64.cpp

```diff
@@ -374,10 +374,7 @@ void LIR_Assembler::jobject2reg_with_patching(Register reg, CodeEmitInfo *info)
 int LIR_Assembler::initial_frame_size_in_bytes() const {
   // if rounding, must let FrameMap know!
 
-  // The frame_map records size in slots (32bit word)
-
-  // subtract two words to account for return address and link
-  return (frame_map()->framesize() - (2*VMRegImpl::slots_per_word))  * VMRegImpl::stack_slot_size;
+  return in_bytes(frame_map()->framesize_in_bytes());
```

Comment:
Umm, really? `framesize_in_bytes()` returns a value that has to be converted to bytes?

Reply:
`framesize_in_bytes()` returns a struct ByteSize that needs to be converted to an int with `in_bytes()`. The extra _in_bytes seems a bit redundant but it's not my name :-)